src/components/alarm/AlarmController.cpp

```diff
@@ -102,14 +98,13 @@ void AlarmController::SetOffAlarmNow() {
 }
 
 void AlarmController::StopAlerting() {
-  systemTask->PushMessage(System::Messages::StopRinging);
-
   // Alarm state is off unless this is a recurring alarm
   if (recurrence == RecurType::None) {
     state = AlarmState::Not_Set;
   } else {
-    state = AlarmState::Set;
     // set next instance
     ScheduleAlarm();
   }
+
```

Comment:
two newlines, I think we only need one line

```suggestion
```

Reply:
I'm indifferent about this. There are no guidelines about this.